setup.py

```diff
@@ -109,9 +112,11 @@
         ],
         'invenio_pidstore.minters': [
             'recid = invenio_pidstore.minters:recid_minter',
+            'recidv2 = invenio_pidstore.minters:recid_minter_v2',
```

Comment:
Minor: There is usually underscore everywhere, so I am not sure if it would be more appropriate `recid_v2`.

Reply:
:heavy_check_mark: